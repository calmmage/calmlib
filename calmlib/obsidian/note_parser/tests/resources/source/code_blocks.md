```python
def hello():
    print("world")
```

```
plain code block
```
