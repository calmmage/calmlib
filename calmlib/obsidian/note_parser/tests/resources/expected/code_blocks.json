{
  "items": [
    {
      "type": "code_block",
      "text": "```python\ndef hello():\n    print(\"world\")\n```",
      "children": [
        {
          "type": "empty",
          "text": ""
        }
      ]
    },
    {
      "type": "code_block",
      "text": "```\nplain code block\n```"
    }
  ]
}